composer.json

```diff
@@ -20,7 +20,7 @@
         "bin/zf-dependency-injection-cache-warmup"
     ],
     "require": {
-        "php": ">=7.3 | >=8.0",
+        "php": ">=8.0",
```

Comment:
I would suggest the same versions like laminas-servicemanager uses: [`"php": "~7.4.0 || ~8.0.0 || ~8.1.0",`](https://github.com/laminas/laminas-servicemanager/blob/3.10.0/composer.json#L27)
And besides that, [PHP does not follow semantic versioning](https://www.php.net/releases/8.1/en.php#deprecations_and_bc_breaks).

Reply:
My idea was to drop php7 support somewhere in the future, but looks good to stick to the service manager requirements because my module relies on him :-D